README.md

```diff
@@ -67,7 +67,49 @@ hasura.user // will be reset to anonymous user
 ### Data query
 
 **NOTE**: In the examples below, `onSuccess` and `onError` are callback functions that you must implement.
+    
+For example:
+    
+**Option 1:**
 
+Use lambdas or anonymous functions directly for handling success/error.
+
+```javascript
+hasura.data.query({
+  type: 'select',
+  args: {
+    table: 'article',
+    columns: ['*']
+  }},
+  (data) => { console.log(data); },
+  (error) => { console.log(error); }
+);
+```
+**Option 2:**
+
+Use predefined functions as shown below:
+
+```javascript
+function mySuccessHandler (data) {
+  console.log(data);
+}
+
+function myErrorHandler (e) {
+  console.log(e);
+}
+
+hasura.data.query({
+  type: 'select',
+  args: {
+    table: 'article',
+    columns: ['*']
+  }},
+  mySuccessHandler,
+  myErrorHandler
+);
+```
+
+The Data Query can be implemented as:
```

Comment:
This portion is also redundant now. "The data query can be implemented as" and the entire code block below it too.